attractors_pick: [-0.704, 0.099, 0.599,
                 -0.597, -0.0811, 0.607]
attractor_place: [-0.690, 0.093, 0.345]

